eureka_ml_insights/user_configs/vision_language/maze.py

```diff
@@ -81,23 +82,27 @@ def configure_pipeline(self, model_config: ModelConfig, resume_from: str = None)
                     "format": ".jsonl",
                     "transform": SequenceTransform(
                         [
+                            ExtractQuestionOptions(
+                                    prompt_column_name="prompt",
+                                    extracted_options_column_name="target_options_answers",
+                            ),
                             ColumnRename(name_mapping={"model_output": "model_output_raw"}),
-                            ExtractAnswerMaze(
+                            ExtractAnswerSpatialMapAndMaze(
                                 answer_column_name="model_output_raw",
                                 extracted_answer_column_name="model_output",
-                                question_type_column_name="question_type",
+                                extracted_options_column_name="target_options_answers",
                             ),
                         ],
                     ),
                 },
             ),
-            metric_config=MetricConfig(CaseInsensitiveMatch),
+            metric_config=MetricConfig(CaseInsensitiveOrMatch),
             aggregator_configs=[
-                AggregatorConfig(CountAggregator, {"column_names": ["CaseInsensitiveMatch_result"], "normalize": True}),
+                AggregatorConfig(CountAggregator, {"column_names": ["CaseInsensitiveOrMatch_result"], "normalize": True}),
```

Comment:
Is there a prompt change that leads to multiple answers instead of just one?

Reply:
It's really more of a function of being more careful in the answer extraction for the current (unchanged) prompts.  I saw the models are sometimes giving the letter answer for an MCQ and sometimes the full word answer.  I wanted to parse both and have the option to consider the answer correct if either one was right (or alternatively we could be more strict and require them to be consistent with each other).